topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md

```diff
@@ -672,6 +673,7 @@ Congratulations! You should have a file with all the unique gene names and a cou
 One of the most important features of Galaxy comes at the end of an analysis. When you have published striking findings, it is important that other researchers are able to reproduce your in-silico experiment. Galaxy enables users to easily share their workflows and histories with others.
 
 To share a history, click on the {% icon galaxy-gear %} (gear) symbol in the history pane and select `Share or Publish`. On this page you can do 3 things:
+![Publish Your Work](../../images/share_work.PNG)
```

Comment:
It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details.

You can 'test' the alt text by just looking at your markdown. Is what you've written sufficient to describe the image if it wasn't there? Could you please check all of the images you've added, I think they all need to be more descriptive.